quarkus/extension/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -3,7 +3,7 @@ name: "QSON"
 metadata:
   keywords:
   - "json"
-  guide: "https://quarkus.io/guides/qson"
+  guide: "https://github.com/quarkusio/qson"
```

Comment:
Should we point to https://github.com/quarkusio/qson#readme instead?

Reply:
Yes, that's better, thanks! Will pop in an update and squash.